### DISCUSSION TITLE
Separating Underlay and Fill to run non-consecutively?

### DISCUSSION THREAD

**Discussion by jonnierose** [inkstitch/inkstitch#2563] [Q&A]
Is it currently possible to run an underlay for an object, then fill or satin stitch a totally different object, and then return to the first object to do the proper fill stitch?
I currently cheat to do something similar by duplicating the object I want to do this with and trying to get the fill settings as close to the proper underlay as I can (though I can't seem to get it all that close) and then having the regular fill with underlay ticked off when I need the final layer. 
Basically I'd like to be able to tick the fill box off in params like you can with the underlay. I am not equipped to be a big girl coder, but I'll probably still dig around to see if I can break something just enough to do this. 
TIA, I love you all. 
*edit: grammar

**Comment by tyrosinase** [comment]
I've been known to build a stitch preview layer, then grab the stitched-out object and split it into two separate manual-stitch-placement objects (saving the "source" objects in a hidden layer). Ditto the underlay for satins - the zigzag underlay makes a much more attractive tackdown for clear vinyl (or other non-backed plastics where you don't want so much needlepunch that you make a tear line) than a single zigzag/satin stitch. (Though hmm, maybe you can get that with two reversed-rails satins these days? I haven't tried.)

Anyway, seconding the motion for that feature. Another option box! :laughing:

**Reply by jonnierose** [reply]
OooOoh that's smart, I might steal that :)
